readme.txt

```diff
@@ -214,6 +214,15 @@ Yes, through the addition of one or more of the add-on payment gateways, you can
 
 == Changelog ==
 
+= 2.6.3, June 30, 2016 =
+
+* Fix: Product name needs to be trimmed during payment import to prevent duplicate products being created
+* Fix: Discount codes getting erroneously marked as Inactive
```

Comment:
I think `erroneously` isnt' the right word. Maybe `incorrectly marked as Inactive during nightly cleanup routine`.